build/project-template-gradle/src/main/java/com/tns/RuntimeHelper.java

```diff
@@ -114,21 +115,29 @@ public void initRuntime()
 				}
 			}
 			
-			
-
-			
 			runtime.init();
 			runtime.runScript(new File(appDir, "internal/ts_helpers.js"));
-
-			File javaClassesModule = new File(appDir, "app/tns-java-classes.js");
```

Comment:
Can we get this back to the project template? It is needed by the android-snapshot plugin and by the webpack plugin.